docs/mounting.md

```diff
@@ -31,7 +31,14 @@ When mounting with an fstab entry, use the key_file option:
 
     my-bucket /mount/point gcsfuse rw,noauto,user,key_file=/path/to/key.json
 
-When you create a Compute Engine VM, its service account can also be used to authenticate access to Cloud Storage FUSE. When Cloud Storage FUSE is run from such a VM, it automatically has access to buckets owned by the same project as the VM.
+When you create a Compute Engine VM, its service account can also be used to
```

Comment:
Bonnie has raised a PR to update mounting doc to cloud docs. Please update the statement in cloud doc itself.

https://github.com/GoogleCloudPlatform/gcsfuse/pull/1182/files